src/_includes/partials/head.njk

```diff
@@ -8,6 +8,7 @@
 	<title>{{ title }} - {{ metaTitle }}</title>
 	<meta content="width=device-width,initial-scale=1" name="viewport">
 	<meta name="description" content="{{ description }}">
+	<meta name="robots" content="none">
```

Comment:
This meta name=robots should contains a valid value according to HTML 5 spec
ref.
* http://www.robotstxt.org/meta.html
* https://wiki.whatwg.org/wiki/MetaExtensions

Reply:
FYI - "none" is might only supported by google bot, using "noindex, nofollow" might be more cross-bot compatible